### PR TITLE
Update engine-api to 1fb8f09960cc32b9648495a422c960fb2a4f8a09

### PR DESCRIPTION
**Pull request by mrunalp**
Update engine-api for https://github.com/docker/docker/pull/22481/files

Signed-off-by: Mrunal Patel mrunalp@gmail.com

### CODE REVIEW

**Comment by mrunalp** [comment]
@LK4D4 PTAL

**Comment by mrunalp** [comment]
I need to fix up code. Looking into it.

**Comment by mrunalp** [comment]
okay, pushed update that should fix the build.

File: api/client/search.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func (cli *DockerCli) CmdSearch(args ...string) error {
 
 		}
 		fmt.Fprint(w, "\t")
-		if res.IsAutomated || res.IsTrusted {
+		if res.IsAutomated || res.IsOfficial {
```
Comment (cpuguy83) [inline api/client/search.go]: I don't think this should be `IsOfficial`. `IsTrusted` was the same as `IsAutomated` before.

Comment (mrunalp) [inline api/client/search.go]: @cpuguy83 What is the right fix? Just to remove res.IsTrusted?

Comment (cpuguy83) [inline api/client/search.go]: Seems so.

Comment (mrunalp) [inline api/client/search.go]: @cpuguy83 Updated.

**Comment by cpuguy83** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Pull request merged**